### DISCUSSION TITLE
Fundamentos Entity Framework - No DbContext was found in assembly

### DISCUSSION THREAD

**Discussion by mion24** [balta-io/forum#349] [Tira Dúvidas] [answered]
bom dia, 

estou tendo esse problema ao tentar utilizar o comando add-mgiration "No DbContext was found in assembly 'ef'. Ensure that you're using the correct assembly and that the type is neither abstract nor generic."

estava tendo problemas p utilizar o console powershell e passei a utilizar o console do gerenciador de pacotes, vou enviaro print de como está minha classe com o DBContext e todas as informações. pode me dar uma ajuda por favor ?
![imagem_2023-06-19_105709644](https://github.com/balta-io/forum/assets/104873936/e68e663a-e50b-48b8-8066-f4051d316844)

**Comment by mion24** [comment, marked as answer]
já resolvi, aparentemente o problema ocorria por conta do nome do projeto, que era uma palavra reservada do entity, se atentem a essa questão caso venham a sofrer da mesma dúvida que eu :/

**Reply by andrebaltieri** [reply]
Show!!!
